ra_aid/tools/expert.py

```
@@ -13,13 +15,57 @@
+
+
 def get_model():
     global _model
     try:
         if _model is None:
-            config = _global_memory["config"]
-            provider = config.get("expert_provider") or config.get("provider")
-            model = config.get("expert_model") or config.get("model")
+            provider = _global_memory["config"]["expert_provider"] or "openai"
+            model = _global_memory["config"]["expert_model"] or "o1"
+            auto_select_expert_model = _global_memory["config"]["expert_auto_select_model"] or False
```

Ruff (0.8.2)

Local variable `auto_select_expert_model` is assigned to but never used

Remove assignment to unused variable `auto_select_expert_model`

(F841)